docs/en/tutorials/customize_datasets.md

```diff
@@ -1,5 +1,181 @@
 # Tutorial 2: Customize Datasets
 
+## How to prepare your own dataset
+
+Preparation of your own dataset is divided into following steps:
+
+- Dataset preparation, generating correct data format and saving it to correct place like `./data/`.
+- Define your dataset in `./mmseg/datasets/` to register it in `DATASETS`.
+- Setting hyperparameters of training and validation in `./configs/_base_/datasets/`, such as data augmentation strategy, image crop size and image pixel normalization.
+
+Files needed to be modified are listed below:
+
+```none
+mmsegmentation
+   |
+   |- data
+   |     |- my_dataset                 # your dataset after data conversion
+   |- mmseg
+   |     |- datasets
+   |     |     |- __init__.py          # add your dataset class here
+   |     |     |- my_dataset.py               ## define your dataset class
+   |     |     |- ...
+   |- configs
+   |     |- _base_
+   |     |     |- datasets
+   |     |     |     |- my_dataset_config.py      # config of your dataset
+   |     |     |- ...
+   |     |- ...
+   |- ...
+```
+
+### Preparation of dataset
+
+First, convert your dataset into following format, where `img_suffix` and `seg_map_suffix` are format of images and annotations, which are usually `.png` and `.jpg`.
+
+```none
+├── data
+│   ├── my_dataset
+│   │   ├── img_dir
+│   │   │   ├── train
+│   │   │   │   ├── xxx{img_suffix}
+│   │   │   │   ├── yyy{img_suffix}
+│   │   │   ├── val
+│   │   │   │   ├── zzz{img_suffix}
+│   │   ├── ann_dir
+│   │   │   ├── train
+│   │   │   │   ├── xxx{seg_map_suffix}
+│   │   │   │   ├── yyy{seg_map_suffix}
+│   │   │   ├── val
+│   │   │   │   ├── zzz{seg_map_suffix}
+
+```
+
+### Dataset Registration
+
+After your dataset conversion done above, create a new file `my_dataset.py` in `./mmseg/dataset/`, where defining `MyDataset` class could be registered into `DATASETS` of MMCV and be used by model.
+
+
+```python
+from .builder import DATASETS
+from .custom import CustomDataset
+
+# Register MyDataset class into DATASETS
+@DATASETS.register_module()
+class MyDataset(CustomDataset):
```

Comment:
Might explain the main interfaces and their function of CustomDataset

Reply:
> 
In my opinion, it is better to help them run the code, as for explanations of advanced class of parameters like `CustomDataset` and `reduce_zero_label`, I wonder if it pushes too much strength on beginners who are totally new to this codebase.